README.md

```diff
@@ -15,7 +15,7 @@ You can also specify Julia version you want to use in [`Project.toml`](https://j
 
 To run the server for your project you can use something like
 ```
-web: julia --project 'using Foo; Foo.serve($PORT)'
+web: julia --project -e 'using Foo; Foo.serve($PORT)'
```

Comment:
We will need to change the [sample](https://github.com/Optomatica/heroku-julia-sample) to support this approach. 

The earlier convention assumes a different way to "package" the web server. Instead of starting a custom project, we simply use `Mux.jl` as a wrapper around a core custom built package. The thinking was that this is probably a light weight wrapper. 

I could see the advantages to following the more conventional way and having a "proper" project the exports certain methods etc... 

I think a good strategy is to present the user with both possibilities, but we will need to expand the documentation with sample code accordingly.

Reply:
> "proper" project the exports certain methods etc...

Yeah exactly. The method exporting is nice, but also things like documentation, testing, CompatHelper.

Sorry for taking a while to reply to your good comments. I've tried to address them in https://github.com/Optomatica/heroku-buildpack-julia/pull/9/commits/317847b4fe4f6f14faa366f5face1905bb8e36b5 by making my changes non-breaking.